modules/persistenceService/src/test/scala/config/DbConfigSpec.scala

```diff
@@ -10,7 +10,16 @@ class DbConfigSpec extends AsyncFreeSpec with AsyncIOSpec with Matchers {
     "loadApplicationConfig" in {
       ApplicationConfig
         .loadApplicationConfig[IO]("dummy.jdbc")
-        .map(result => assert(result.isRight))
+        .map(config =>
+          assert(
+            config == ApplicationConfig(
+              DriverName("org.postgresql.Driver"),
+              ConnectionUrl("jdbc:postgresql:dummyproject"),
+              User("sabasimi"),
+              PassWord(Some(""))
+            )
+          )
+        )
```

Comment:
What it's the reason for restoring this?

Reply:
We were using `isRight` in the test because we were returning and `Either` with an `application config`.
But now we are are returning an `F[ApplicationConfig`] which means `isRight` can't be used.

We need to check the content of the config.

Reply:
I see, there's another option then, use the handy `.attempt`

```scala
ApplicationConfig
  .loadApplicationConfig[IO]("dummy.jdbc")
  .attempt
  .map(result => assert(result.isRight))
```

Ref: [MonadError#attempt](https://typelevel.org/cats/api/cats/MonadError.html#attempt[A](fa:F[A]):F[Either[E,A]])

Reply:
We don't care about the values in the config; we want to be sure that our scala models map to the config files.

Reply:
sure, I will make the necessary changes.
thank you